superchain/configs/sepolia/opdeployer.toml

```diff
@@ -0,0 +1,60 @@
+name = "op-deploy-sepolia-test"
+chain_id = 12345
+public_rpc = "http://awe.some.rpc"
```

Comment:
Looks like most of your validation test failures (which are now skipped/excluded) are related to this rpc url, which cannot be called to retrieve info from the L2 chain

Reply:
okay cool, we don't actually have an l2 api for this testnet. So I think it makes sense to skip these.